app/views/dashboard/index.html.erb

```diff
@@ -27,42 +26,32 @@
       <% else %>
         <ul>
            <% current_user.medications.each do |medication| %>
-             <li><%= medication.brand_name %></li>
+             <li><%= medication.brand_name %>
```

Comment:
was the closing `</li>` intended to be removed here?

Reply:
Fixed!